### ISSUE TITLE
Adapt first notebooks with KNearestNeighbors / LogisticRegression

### ISSUE THREAD

**Issue by lesteve** [INRIA/scikit-learn-mooc#110]
The idea is to have 
- a first simple notebook with KNearestNeighbors as done by @GaelVaroquaux i.e. separate csv for train and test with numerical-only features
- a second notebook as we had with full dataset + pandas slicing to select numerical feature + `train_test_split` LogisticRegression warning
- add a few words about LogisticRegression to give some simple intuition like `0.1 * age + 3.3 * education-num > 0.5`
- an exercise with n_neighbors=1 100% training accuracy
- also #108 to say that KNearestNeighbors is used to be didactic and may not be very useful in practice. Probably not worth putting it in the notebook but I am also curious about the accuracy of `KNearestNeighbors` compared to the `DummyClassifier` to be honest.

**Comment by GaelVaroquaux** [comment]
Can we break down this plan by teaching objective.

What is the teaching objective of each notebook?

**Comment by lesteve** [comment]
Good point, I'll try to think about this. I was thinking maybe about taking the plan.md and updating it with what we have so far but this takes some time so we will see. The idea would be to allow a better overview of where we stand so far. At one point we thought the general table of contents https://inria.github.io/scikit-learn-mooc/ could provide this, but I think we need more than this ...

By the way something that could be of general interest (done by Software Carpentry people) about lesson design:
http://teachtogether.tech/en/index.html#review-2

**Comment by glemaitre** [comment]
I push a new notebook. In terms of objective:

* notebook 1: first model in scikit-learn -> API + necessity train/test split (overfitting)
* notebook 2: identify numerical columns, select columns, use train_test_split when all data are loaded, see a model which is usually more in use than kNN
* notebook 3: identify issue by not preprocessing numerical data
* notebook 4: identify categorical columns, show preprocessing for these type of data
* notebook 5: merge both numerical and categorical

**Comment by glemaitre** [comment]
We will need to have more exercises.
